### ISSUE TITLE
cpu temp for bananapi

### ISSUE THREAD

**Issue by BOFHELL** [StefanCohen/OctoPrint-Dashboard#139]
I use Dasboard on a BananaPi, to show the cpu temp i add it manually.
to be safe for the next update i hope u can append it in your code

psutils returned  

> {'soc_thermal': [shwtemp(label='', current=0.056, high=1.2e-10, critical=1.2e-10)]}

so i add this code
```
        elif 'soc_thermal' in thermal:
            self.cpu_temp=int(round(float(thermal['soc_thermal'][0][1])*1000))
```

thx

**Comment by StefanCohen** [comment]
Thanks. I will take a look.

//Stefan

**Comment by StefanCohen** [comment]
Ok. I have no way to verify this but I have added the check to 1.12.0. Please verify the implementation

**Comment by StefanCohen** [comment]
Fixed in 1.12.0
